src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -185,6 +185,19 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <!--
+    Check for improper casing of the 'RuntimeIdentifier' value.
+    For reasons such as case-sensitive file system, we prefer lower-case 'RuntimeIdentifier' value.
+  -->
+  <Target Name="_CheckForRuntimeIdentifierCasing"
+          BeforeTargets="_CheckForInvalidConfigurationAndPlatform;RunResolvePackageDependencies;GetFrameworkPaths;GetReferenceAssemblyPaths;Restore"
+          Condition="'$(AppendRuntimeIdentifierToOutputPath)' == 'true' and '$(RuntimeIdentifier)' != '' and '$(_UsingDefaultRuntimeIdentifier)' != 'true'">
+
+    <NETSdkError Condition="!$(RuntimeIdentifier.Equals('$(RuntimeIdentifier.ToLowerInvariant())'))"
```

Comment:
If we include this I think it should be a warning, not a error.

Reply:
Yeah!! My mistake.